start-from-zero/evaluation-zero.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Evaluation</title>
+    <link rel="stylesheet" href="https://necolas.github.io/normalize.css/7.0.0/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    
+    <nav class="menu">
+        
+         <div class="container" id='height-100'>
+           <div class="row" id='height-100'>
+               <div class="col-sm-12 position-div" id='height-100'>
+                   <div class="one"><a href="#0">Evaluations</a></div>
+                   <div class="two"><a href="new-evaluation-zero.html">New Evaluations</a></div>
+               </div>
+           </div>
+
+         </div>
+        
+        <ul class='down-ul'>
+            <li><a href="index.html"><img src="assets/126-Softvision-logo.png"
+             alt=""></a></li>
+             <li id='logout'>Logout</li>
+        </ul>
+    </nav>
+    
+    <section class="table">
+        <div class="container">
+            <div class="row-flex1">
+                <table>
+                    <thead>
+                        <tr id="table-row">
+                            <th>Nume</th>
+                            <th>Tehnologie</th>
+                            <th>Nivel</th>
+                            <th></th>
+                        </tr>
+                    </thead>
+                    <tbody id='tbody'>
+                        <tr>
+                            <td>Popescu Adrian</td>
+                            <td>Javascript</td>
+                            <td>Mid 2</td>
+                            <td class='detalii'>Detalii 
+                                <div><span><a class='plus-sign' href="#">+</a></span></div>
```

Comment:
Any particular reason for the `<a>` to be wrapped in a `<span>`, itself wrapped in a `<div>`? Does it bring added value to the table? Can we do without the extra elements? Also, would there be other, more appropriate tags that we could use for what looks like the Add button? I'd also argue that the look and feel of the button would be best tackled via CSS, instead of in HTML, with a hard-coded `+` symbol. At some point in the future, we might want the button to say "Add" on desktop, but only keep the "+" character on mobile, to accommodate for the smaller screen size.